react/ProductContextProvider.js

```diff
@@ -119,6 +119,11 @@ class ProductContextProvider extends Component {
       product,
     }
 
+    const breadCrumbsProps = {
```

Comment:
add a doc for this decision, and `breadcrumbsProps` is better

Reply:
👍